.github/workflows/build_workflow.yml

```diff
@@ -126,6 +126,8 @@ jobs:
       - name: Build Sphinx Docs
         run: |
           cd docs
+          python source/quickguides/generate_quick_guides.py
```

Comment:
@tomvothecoder If I run this command locally and then run `make html`, I can generate the proper documentation, but if I run `sphinx-multiversion source _build/html` instead, none of my changes seem to take effect.

This also seems to be the case in my test https://github.com/forsyth2/e3sm_diags/pull/2. After merging that pull request to my fork's `master`, the `gh-pages` doesn't have the new quick guides (see https://github.com/forsyth2/e3sm_diags/tree/gh-pages/_build/html/master/quickguides)

It doesn't look like Sphinx is finding the generated `rst` files -- do they need to be added to `git` perhaps? (see https://github.com/forsyth2/e3sm_diags/runs/3306348047?check_suite_focus=true#step:6:63)

Reply:
Yes, the quick guide `.rst` files need to be pushed because `sphinx-multiversion` uses those source files within each branch/tag for the HTML builds. You probably need to add a separate step to generate quick guides and push them to the repo before `Build Sphinx Docs`.

Refer to lines below for pushing quick-guide `.rst` files to `master` (the `publish-docs` job should only run on `master`). 
https://github.com/E3SM-Project/e3sm_diags/blob/27a12f93b75052ef2f9b62315e13afb3d034c8c2/.github/workflows/build_workflow.yml#L148-L154

Reply:
> You probably need to add a separate step to generate quick guides and push them to the repo

Does that mean a pull request to `master` that updates the quick guides will not only include the commit from merging the pull request but also a subsequent GitHub actions commit for the generated `rst` files?

Reply:
Yes, this is one issue related with attempting to generate quick guides dynamically using a separate python script.

The other way is to manually run the python script and push the `.rst` files. This might end up being the better solution if the quick guides aren't updated that much, and attempting to automate it doesn't provide that much value.